src/main/resources/atlassian-plugin.xml

```diff
@@ -38,6 +38,7 @@
     <context>jira.view.issue</context>
     <context>atl.general</context>
     <dependency>com.atlassian.auiplugin:ajs</dependency>
+    <dependency>com.atlassian.auiplugin:aui-select2</dependency>
```

Comment:
why do you need this?

Reply:
I need the select2 in the coverage panel to use it as the dropdown